services/src/main/java/org/keycloak/authentication/authenticators/browser/ScriptBasedAuthenticatorFactory.java

```diff
@@ -153,6 +142,6 @@ public List<ProviderConfigProperty> getConfigProperties() {
 
     @Override
     public boolean isSupported() {
-        return Profile.isFeatureEnabled(Profile.Feature.SCRIPTS) && Profile.isFeatureEnabled(Profile.Feature.UPLOAD_SCRIPTS);
+        return Profile.isFeatureEnabled(Profile.Feature.SCRIPTS);
```

Comment:
@stianst @pedroigor AFAIK this means that rest-based provider `ScriptBasedAuthenticator` will be now enabled when "scripts" feature is enabled. Is it really expected behaviour? Should not this method always return `false` now to make sure provider is not available anywhere?

Note that I've did just very quick scan of this PR and I might be missing some details. Just wanted to doublecheck if this is expected or rather a mistake...

Reply:
The provider is no longer registered as a service, that is why I did not change here.

Reply:
Yes, true. Thanks for pointing this!